tests/test_visibility.py

```diff
@@ -53,7 +53,7 @@ def test_filtering(self):
         #
 
         # before we do that, we persist some other events to act as state.
-        self.inject_visibility("@admin:hs", "joined")
+        yield self.inject_visibility("@admin:hs", "joined")
```

Comment:
Use `self.get_success` instead so we don't need to update in future?

Reply:
Though I do see this is consistent with other lines in this method...

Reply:
It is keeping it consistent with the rest of the test-case, yes.

Reply:
(I think I said it last week, but updating all the tests is a separate task IMO.)